src/pattern-library/components/patterns/prototype/SharedAnnotationsPage.tsx

```diff
@@ -240,8 +241,8 @@ export default function SharedAnnotationsPrototypePage() {
                 <FakeAnnotation isOwn />
               </FakeSidebar>
             </Library.Demo>
-          </Library.Example>
-        </Library.Pattern>
+          </Library.Section>
+        </Library.Section>
```

Comment:
I'm not entirely convinced that nested `Section` components will be any less fiddly to work with than section-components-with-unique-names, but we'll see.